library/src/blas1/reduction_strided_batched.h

```diff
@@ -183,13 +182,15 @@ template <rocblas_int NB,
           typename REDUCE = rocblas_reduce_sum,
           typename TPtrX,
           typename To>
-__global__ void rocblas_reduction_strided_batched_kernel_part1(rocblas_int    n,
-                                                               rocblas_int    nblocks,
-                                                               TPtrX          xvec,
-                                                               rocblas_int    shiftx,
-                                                               rocblas_int    incx,
-                                                               rocblas_stride stridex,
-                                                               To*            workspace)
+__attribute__((amdgpu_flat_work_group_size((NB < 128) ? NB : 128, (NB > 256) ? NB : 256)))
```

Comment:
This should have a comment as it isn't obvious why it is here, and how generally applicable it is.